modules/driver-mongo/src/main/kotlin/datamaintain/db/driver/mongo/MongoDriver.kt

```diff
@@ -19,16 +20,16 @@ class MongoDriver(dbName: String,
                   private val tmpFilePath: Path
 ) : DatamaintainDriver {
     private val database: MongoDatabase
-    private val executedScriptsCollection: MongoCollection<ScriptWithoutContent>
+    private val executedScriptsCollection: MongoCollection<Document>
 
     companion object {
         const val EXECUTED_SCRIPTS_COLLECTION = "executed-scripts"
     }
 
     init {
-        val client = KMongo.createClient(mongoUri)
+        val client = MongoClients.create("mongodb://$mongoUri")
```

Comment:
In addition to the remark of @Lysoun in AbstractMongoDbTest I wonder if mongoUri is well-named. Maybe mongoHost is better.